CMakeLists.txt

```diff
@@ -78,8 +80,8 @@ else ()
         file(GLOB AWS_CAL_OS_SRC
             "source/unix/*.c"
         )
-        find_package(LibCrypto REQUIRED)
-        set(PLATFORM_LIBS LibCrypto::Crypto dl)
+        find_library(AWS_LC crypto REQUIRED)
+        set(PLATFORM_LIBS dl ${AWS_LC})
```

Comment:
trivial: What's more normal for Cmake? To define a variable, or just have a normal fixed string?

Reply:
Well, these are different. One is finding a package and a named target, one is finding a library and storing it in a variable, and then the expanded path is added to the link line.

Reply:
Sorry, I was not clear. I meant: is it normal for a find_library() call to have the side-effect of setting a variable `${AWS_LC}` that you then pass to PLATFORM_LIBS? Or is it more normal to just have the user type out both strings
```
find_library(AWS_LC crypto REQUIRED)
set(PLATFORM_LIBS dl LibCrypto::Crypto) # or whatever ${AWS_LC} expands to
```